packages/liquidator/test/Liquidator.js

```diff
@@ -42,640 +33,295 @@ const configs = [
   { tokenSymbol: "BTC", collateralDecimals: 8, syntheticDecimals: 18, priceFeedDecimals: 8 },
   { tokenSymbol: "BTC", collateralDecimals: 8, syntheticDecimals: 8, priceFeedDecimals: 18 }
 ];
-// allows this to be set to null without throwing.
```

Comment:
the test changes made here are very similar to the other two sets. See the new tests at the bottom that test the perp specific logic.